interface.go

```diff
@@ -131,13 +144,33 @@ func (s *BaseGlobalLookupFactory) ZonefileInput() bool {
 // keep a mapping from name to factory
 var lookups map[string]GlobalLookupFactory
 
+// keep a mapping from name to input handler
+var inputs map[string]InputHandler
```

Comment:
Since this is shared throughout the zdns module, maybe rename to `inputHandlers`/`outputHandlers` or something even more specific?